contrib/depdot.sh

```diff
@@ -10,7 +10,7 @@
 shopt -s nullglob
 
 
-pushd "$(dirname "$0")/.." >/dev/null
+pushd "$(dirname "$0")/.." >/dev/null || exit
```

Comment:
Let's exit with non-zero exit status on error.

Reply:
In which case `set -e` would suffice.

Reply:
@mweinelt shellcheck still recommend the exit condition wether or not set -e is added.

Reply:
@mweinelt applied